jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -85,6 +90,21 @@ def __init__(self, *args, **kwargs):
         self.subset_states = []
         self.spectral_display_unit = None
 
+        link_type = getattr(self.app, '_link_type', None)
```

Comment:
This attribute always exists now.
```suggestion
        link_type = self.app._link_type
```

Reply:
i changed this back, doing something like:
`cubeviz_helper = Cubeviz()`
`data = Data(flux=np.ones((128, 128, 256)), label='Test 3D Flux')`
`cubeviz_helper.app.data_collection.append(data)`
was raising an attribute error (even though eventually when the data is loaded it will have a link type of 'pixels', it seems like its running something in the subset plugin before it gets to there)